src/nft/components/collection/CollectionStats.tsx

```diff
@@ -267,8 +269,8 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
         </StatsItem>
       ) : null}
       {stats.stats?.total_listings ? (
-        <StatsItem label="Listings" isMobile={isMobile ?? false}>
-          {totalListingsStr}
+        <StatsItem label="Listed" isMobile={isMobile ?? false}>
+          {listedPercentageStr} %
```

Comment:
I think we should get rid of the space before the `%` which would be in parity with our percentage behaviour elsewhere on the app